apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go

```diff
@@ -269,6 +269,35 @@ type InMemoryDBConfig struct {
 	PVC bool `json:"pvc,omitempty"`
 }
 
+// Describes what amount of app config is mounted to the pod
+// +kubebuilder:validation:Enum={"none", "app", "", "environment"}
+type ConfigAccessMode string
+
+type TestingConfig struct {
+	// Defines the environment for iqe/smoke testing
+	Iqe IqeConfig `json:"iqe,omitempty"`
+
+	// The mode of operation of the IQE Pod. Valid options are:
```

Comment:
Technically now `The mode of operation of the testing Pod` :)

Reply:
I almost had it all fixed lol. Updated.